.github/workflows/go.yml

```diff
@@ -78,4 +78,4 @@ jobs:
       - name: Run Gosec Security Scanner
         uses: securego/gosec@master
         with:
-          args: '-exclude-generated  ./...'
```

Comment:
Wouldn't `-no-fail` ignore scan failures? I don't think we want that.

Reply:
I see in the logs of that action that it failed, but the action is green:
```
 Golang errors in file: []:
  > [line 0 : column 0] - error obtaining VCS status: exit status 128
	Use -buildvcs=false to disable VCS stamping.
```

Reply:
Yep, it won't fail but it will scan. I posted in slack my take on this. Waiting for feedback on how we should address this. Just wanted to prove this was an option.